process/coordinator/process.go

```diff
@@ -479,14 +479,14 @@ func (tc *transactionCoordinator) CreateMbsAndProcessCrossShardTransactionsDstMe
 
 		preproc := tc.getPreProcessor(miniBlock.Type)
 		if check.IfNil(preproc) {
-			log.Error("no preprocessor for miniblock found", "type", miniBlock.Type)
-			continue
+			log.Warn("no preprocessor for miniblock found", "type", miniBlock.Type)
```

Comment:
we might remove this log completely as the error goes all the way up till syncblock or consensus. We might thing that a malicious node deliberately include a miniblock with an invalid block type, causing log warn prints on all other nodes. Might wrap the error though:
```GO
return nil, 0, false, fmt.Errorf("%w unknown block type %d", process.ErrNilPreProcessor, miniBlock.Type)
```

Reply:
fixed